server/test/new.test.ts

```diff
@@ -0,0 +1,8 @@
+import { describe, test, expect } from '@jest/globals';
```

Comment:
Rename this file to `preliminaries.test.ts`

Reply:
File changes were done.

Reply:
i am converting this pr to draft, we will modify the babel config and get back.

Reply:
Ahmed and I resolved the errors by creating the necessary config files and installing the required dependencies.
![Screenshot (152)](https://github.com/user-attachments/assets/15113ffa-466b-4ad7-a840-b369d3011eb3)